### PR TITLE
Mark signal glow material for not loosing it.

### PR DESCRIPTION
**Pull request by pzgulyas**
I wonder if this wasn't it missing for you guys. :-) (Hopefully the PR is now to the right branch.)

### CODE REVIEW

File: Source/RunActivity/Viewer3D/Signals.cs [hunk 1]
```diff
@@ -413,6 +413,7 @@ void renderEffect(Material material)
             internal void Mark()
             {
                 SignalTypeData.Material.Mark();
+                SignalTypeData.GlowMaterial?.Mark();
```
Comment (YoRyan) [inline Source/RunActivity/Viewer3D/Signals.cs:416]: Is there any reason to guard against null here? The glow material isn't loaded from the route data.

Comment (pzgulyas) [reply Source/RunActivity/Viewer3D/Signals.cs:416]: In the constructor below this line there is an if-else condition, and the GlowMaterial isn't set in both cases.

Comment (YoRyan) [reply Source/RunActivity/Viewer3D/Signals.cs:416]: Gotcha

**Review by YoRyan — APPROVED** [review]

**Comment by Sharpe49** [comment]
You forgot a part of the OR workflow: 
Is there a bug in Launchpad resolved by this? A blueprint? A Trello item?

**Comment by YoRyan** [comment]
Definitely just a bug, but it hadn't filed on Launchpad. Oops. I guess I should have asked for a report, but I deferred to Peter's judgment.

**Pull request merged**